packages/babel-core/src/api/browser.js

```diff
@@ -1,112 +0,0 @@
-/* eslint max-len: 0 */
```

Comment:
There is also a reference to this file in `package.json` that we should delete now.

It'd also be great to move `api/node.js` to be `index.js` now that we don't have the split entry point.